units/event_log_tests.cc

```diff
@@ -274,4 +373,122 @@ TEST(Event, CreateGraphicsPipelinesEventCreation) {
 
   EXPECT_EQ(EventToCSVString(pipeline_event), expected_str.str());
 }
+
+TEST(EventLogger, MockObjectCreation) {
+  TestLogger test_logger;
+  EXPECT_FALSE(test_logger.IsStarted());
+  EXPECT_EQ(0, test_logger.GetFlushCount());
+  EXPECT_FALSE(test_logger.IsFinished());
+  EXPECT_THAT(test_logger.GetEvents(), testing::IsEmpty());
+}
+
+TEST(EventLogger, MockObjectFunctionCalls) {
+  const int64_t timestamp_val = 1601314732230797664;
+  const int64_t hash_val1 = 0x67d6fd0aaa78a6d8;
+  const int64_t hash_val2 = 0x67d390249c2f20ce;
+  const int64_t duration = 926318;
```

Comment:
Because exact numbers don't matter much in these tests, you can pick some short constants like 1, 2, 3, etc. and use them directly without local variables

Reply:
Same in the tests below